package.json

```diff
@@ -38,7 +38,7 @@
     "test.jest": "jest --silent",
     "test.karma": "karma start",
     "prepare": "husky install",
-    "release": "yarn workspaces run semantic-release -e semantic-release-monorepo"
+    "release": "HUSKY=0 yarn workspaces run semantic-release -e semantic-release-monorepo"
```

Comment:
do we run this command on Windows?
if so, this won't work and we may need to use `cross-env` or something similar.